experiments/exp1337_scaling_suite.py

```diff
@@ -0,0 +1,136 @@
+"""
+Creates a suite of training runs of different model sizes and number of tokens trained on the same data ("modernized Pythia").
+"""
+
+from dataclasses import dataclass, replace
+
+from experiments.dclm.tokenize_dclm import dclm_mixture_config_llama3
+from experiments.defaults import default_train
+from experiments.llama import (
+    compute_num_parameters,
+    llama3_tokenizer_vocab_size,
+    llama_1_4b,
+    llama_1_4b_train_config,
+    llama_8b,
+    llama_8b_train_config,
+    llama_32b,
+    llama_70b,
+    llama_150m,
+    llama_150m_train_config,
+    llama_300m,
+    llama_300m_train_config,
+)
+from experiments.simple_train_config import SimpleTrainConfig
+from experiments.tootsie.exp859_big_tootsies import nemotron_mix
+from levanter.models.llama import LlamaConfig
+from marin.execution.executor import executor_main
+from marin.processing.tokenize.data_configs import LMMixtureDatasetConfig
+from marin.resources import TpuPodConfig
+
+# Like the one in exp859_big_tootsies but with a constant batch size
+llama_32b_train_config = SimpleTrainConfig(
+    resources=TpuPodConfig(tpu_type="v4-2048", slice_count=1),
+    num_train_steps=1_000_000,  # To override later
+    train_batch_size=8192,
+    weight_decay=0.05,
+    learning_rate=4.2e-4,  # TODO: fix this
+    decay=0.4,
+    ema_beta=0.995,
+    lr_schedule="linear",
+    cycle_length=None,
+    allow_partial_checkpoint=True,
+    steps_per_eval=1000,
+    steps_per_task_eval=10000,
+    z_loss_weight=1e-4,
+)
+
+# Like the one in exp750_tootsie70b but with a constant batch size
+llama_70b_train_config_mk6 = SimpleTrainConfig(
+    resources=TpuPodConfig(tpu_type="v6e-128", slice_count=6),
+    num_train_steps=1_000_000,  # To override later
+    train_batch_size=1536,
+    weight_decay=0.05,
+    learning_rate=3.5e-4,  # TODO: fix this
+    decay=0.4,
+    ema_beta=0.995,
+    lr_schedule="linear",
+    cycle_length=None,
+    allow_partial_checkpoint=True,
```

Comment:
allow_partial_checkpoint is probably not necessary

Reply:
ok, I'll remove